projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/config/unit.ts

```diff
@@ -74,7 +74,7 @@ export const userRolesConfig: MyCompanyConfig = {
   ],
 };
 
-// TODO:#checkout update later my company?       name: 'Shipping Addresses',
+// TODO:#future-checkout update later my company?       name: 'Shipping Addresses',
```

Comment:
Let's move this line above the exact place where we are considering the rename, as it's hard to trace it now.

Reply:
This is just a label that we want to rename? Or is there more to it?

Reply:
More to it, because I didn't fully check My-companny. Will double check 👍